dpnp/dpnp_iface.py

```diff
@@ -241,6 +242,37 @@ def check_supported_arrays_type(*arrays, scalar_type=False):
     return True
 
 
+def check_limitations(order=None, subok=False, like=None):
```

Comment:
it would be better to make a function as internal inside `dpnp_iface_arraycreation.py` instead